Libs/MRML/Core/vtkCodedEntry.cxx

```diff
@@ -0,0 +1,156 @@
+/*=auto=========================================================================
+
+Portions (c) Copyright 2017 Brigham and Women's Hospital (BWH) All Rights Reserved.
+
+See COPYRIGHT.txt
+or http://www.slicer.org/copyright/copyright.txt for details.
+
+=========================================================================auto=*/
+
+#include "vtkCodedEntry.h"
+
+// VTK include
+#include <vtkObjectFactory.h>
+
+// STD include
+#include <sstream>
+
+vtkStandardNewMacro(vtkCodedEntry);
+
+//----------------------------------------------------------------------------
+vtkCodedEntry::vtkCodedEntry()
+: CodingSchemeDesignator(NULL)
+, CodeValue(NULL)
+, CodeMeaning(NULL)
+{
+}
+
+//----------------------------------------------------------------------------
+vtkCodedEntry::~vtkCodedEntry()
+{
+  this->Initialize();
+}
+
+//----------------------------------------------------------------------------
+void vtkCodedEntry::Initialize()
+{
+  this->SetCodingSchemeDesignator(NULL);
+  this->SetCodeValue(NULL);
+  this->SetCodeMeaning(NULL);
+}
+
+;
+
+//----------------------------------------------------------------------------
+void vtkCodedEntry::SetSchemeValueMeaning(const std::string& scheme,
+  const std::string& value, const std::string& meaning)
+{
+  this->SetCodingSchemeDesignator(scheme.c_str());
+  this->SetCodeValue(value.c_str());
+  this->SetCodeMeaning(meaning.c_str());
+}
+
+//----------------------------------------------------------------------------
+void vtkCodedEntry::PrintSelf(ostream& os, vtkIndent indent)
+{
+  Superclass::PrintSelf(os,indent);
+  os << indent << "CodingSchemeDesignator: " << (this->CodingSchemeDesignator?this->CodingSchemeDesignator:"(none)") << "\n";
+  os << indent << "CodeValue: " << (this->CodeValue?this->CodeValue:"(none)") << "\n";
+  os << indent << "CodeMeaning: " << (this->CodeMeaning?this->CodeMeaning:"(none)") << "\n";
+}
+
+//----------------------------------------------------------------------------
+void vtkCodedEntry::Copy(vtkCodedEntry* aType)
+{
+  if (!aType)
+    {
+    return;
+    }
+  this->SetCodingSchemeDesignator(aType->GetCodingSchemeDesignator());
+  this->SetCodeValue(aType->GetCodeValue());
+  this->SetCodeMeaning(aType->GetCodeMeaning());
+}
+
+//----------------------------------------------------------------------------
+std::string vtkCodedEntry::GetAsPrintableString()
+{
+  std::string printable = std::string("(")
+    + (this->CodingSchemeDesignator ? this->CodingSchemeDesignator : "(none)") + ", "
+    + (this->CodeValue ? this->CodeValue : "(none)") + ", "
+    + (this->CodeMeaning ? this->CodeMeaning : "(none)") + ")";
```

Comment:
The convention of putting code meaning in double quotes is always applied in DICOM (as an example, see [this section](http://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.7.6.16.2.html#sect_C.7.6.16.2.11.1.2), so I would definitely suggest just putting it in double quotes. I do not think it makes things harder to read, quite the opposite. 

And as above, since we add support that makes things more consistent with the DICOM way of things, I think it is important to be follow the DICOM conventions.